Core/Panel/Dockerfile

```diff
@@ -0,0 +1,56 @@
+# This is the Dockerfile for the Enthought Edge "Panel" core example.
+# 
+# We use "edm-centos-7" as the base image.  This is a Centos-7 based image
+# which includes EDM.
+#
+# EDM dependencies for the app are brought in via a .zbundle file.  This avoids
+# the need to pass your EDM token and/or a custom edm.yaml into the Dockerfile.
+#
+# We perform a two-stage build, to avoid including the .zbundle in the layers
+# of the published image.
+
+
+# IMPORTANT: Please do not define any of the EDGE_* environment variables, or
+# any of the JUPYTERHUB_* variables, directly in this file. They will be set
+# automatically when running in Edge, or by the "ci" module when running
+# locally.
+
+# First stage
+
+ARG BASE_IMAGE=quay.io/enthought/edm-centos-7:3.4.0
```

Comment:
We could update the version number here, I think (we are past 3.4.0).

Reply:
Everywhere in Edge we use :latest (except for src_proxy which copied the version from here, so I'll update that too).